driver_station/package.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>driver_station</name>
+  <version>0.0.0</version>
+  <description>Driver station based on FRC Driver Station</description>
+
+  <maintainer email="kaelan.ms@uwreact.ca">UW React Software Team</maintainer>
```

Comment:
Either change the email to `maintainers@uwreact.ca` to match the name, or change the name to yours to match the email